### DISCUSSION TITLE
Implementing a Tiredness System in Ink

### DISCUSSION THREAD

**Discussion by ronombao24** [inkle/ink#756] [Writing in ink]
Hello. As the title says, I have been planning on making a tiredness system globally that determines how tired the character is through a conditional statement or tunnelling, due to certain actions.

For instance, if `tiredness` is greater than 40 (41 and above), the character will die in the middle of the story, and a game over will happen.

**Comment by Selsynn** [comment]
What did you already tried? There is several way to do it, what do you want to do?
What is the structure you are trying to follow? 

You can look at this discussion you already opened https://github.com/inkle/ink/discussions/757
 the solution could be exactly the same, instead of having the gameover part you could just check the tiredness and display/divert the player to the right knot
